src/platform/julius.c

```diff
@@ -28,6 +28,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#include "platform/ios/ios.h"
```

Comment:
Includes in alphabetical order please.

Other than that,  @bvschaik what do you think?